example/analysis_options.yaml

```diff
@@ -19,3 +19,11 @@ custom_lint:
     - avoid_unnecessary_setstate
     - double_literal_format
     - avoid_unnecessary_type_assertions
+    - banned_external_code:
```

Comment:
What do you think about renaming this lint rule to `avoid_using_api`?

Reply:
`deprecate_external_dependency`?

Reply:
I'll leave the final decision up to you and your team